ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -45,18 +47,31 @@ def setUp(self):
         # No instrumentation code is here because it is present in the
         # conftest.py file next to this file.
         super().setUp()
-
+        Configuration._instance = None  # pylint:disable=protected-access
```

Comment:
what is this code doing? I'm curious if this is surfacing some bug with configuration that should be fixed in the module itself.

Reply:
ok, looks like it's related to removing or resetting configuration. How about adding a method Configuration.reset() or Configuration.clear()?

Reply:
"Resetting" the configuration object is only necessary for testing, not for production code. This  resetting here needs to be done because each test is to be executed with different configuration values, but the `Configuration` class retains its attributes of `_instance` and `__slots__` after every test is executed. I solved this same problem by doing [this](https://github.com/open-telemetry/opentelemetry-python/pull/583/files#diff-c08a394ef868e9985600a69ff38f000eR27) instead, but what is being implemented here is an equivalent solution.

Reply:
I worry that, with no documented or clear way to reset configuration, there will be various approaches that different test suites use to do so. Since both of these approaches make assumptions or rely on behavior of the Configuration implementation, it will become harder to change, or may break unit tests in the future.

Is there a good rationale to not introduce a method? even if it's just for test code, I think a standardized way of accomplishing a specific behavior is important.

Reply:
Might be good to change the Configuration class to allow this. Probably for a separate PR.

Reply:
Yes, I'll be opening an issue and a PR shortly.